index.html

```diff
@@ -22,6 +22,10 @@
 
 <p>A <a href="/guides-template/">template for 18F Guides sites</a> that is also a complete 18F Guides publishing tutorial.</p>
 
+<h2>18F Slides</h2>
```

Comment:
I am confused about the navigation vs links on that page. Shouldn't everything be in the navigation or list below (but not both?)